release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -0,0 +1,440 @@
+[id="ocp-4-8-release-notes"]
+= {product-title} {product-version} release notes
+include::modules/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+Red Hat {product-title} provides developers and IT organizations with a hybrid cloud application platform for deploying both new and existing applications on secure, scalable resources with minimal configuration and management overhead. {product-title} supports a wide selection of programming languages and frameworks, such as Java, JavaScript, Python, Ruby, and PHP.
+
+Built on {op-system-base-full} and Kubernetes, {product-title} provides a more secure and scalable multi-tenant operating system for today's enterprise-class applications, while delivering integrated application runtimes and libraries. {product-title} enables organizations to meet security, privacy, compliance, and governance requirements.
+
+[id="ocp-4-8-about-this-release"]
+== About this release
+
+// TODO: Update k8s link once there is a version-specific URL for the Kubernetes release
+{product-title} (link:https://access.redhat.com/errata/RHSA-2020:5633[RHSA-2020:5633]) is now available. This release uses link:https://kubernetes.io/docs/setup/release/notes/[Kubernetes 1.20] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
```

Comment:
Should this RHSA errata link be updated to something generic, or with a TODO so that we know it needs to be updated?